custom_components/sector/lock.py

```diff
@@ -41,6 +41,8 @@ async def async_setup_entry(
     else:
         _LOGGER.debug("No lock entities to add.")
 
+    _LOGGER.debug("Sector Lock: Firing sector_alarm_lock_setup_complete event to notify event.py")
```

Comment:
Does it really make sense to mix in stuff we have entities for?
Can we not limit the event entity to only be things for which there are not entities?